trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/getLatestFacilityEndDateValues.test.ts

```diff
@@ -37,6 +38,7 @@ describe('getLatestFacilityEndDateValues', () => {
           { ...MOCK_AMENDMENT, updatedAt: 1723653111, version: 1 },
           { ...MOCK_AMENDMENT, updatedAt: 1723653222, version: 2 },
         ],
+        tfm: {},
```

Comment:
I think this can be removed since `facility` (line 8/35) has a TFM object?

Reply:
Ah whoops, thanks!

Reply:
Can't see this being removed @RLavender98 - not sure if it's a Github issue. Happy for this to be done in the next PR since it's very minor.